### ISSUE TITLE
Support for custom extensions

### ISSUE THREAD

**Issue by quizout** [Visual-Stylecop/Visual-StyleCop#33]
It would be nice if this had a better way of dealing with custom extensions.  In order to get a custom extension to work you have to drop the extension dll in the install folder.  With VS extensions it's difficult to identify the location and it changes with every update.  It would be nice if StyleCop extensions could be placed in a more general location.

**Comment by ptittof57** [comment]
You are right, I'll try to do my best to provides a standalone package like previous version but this is not the priorities for now.

**Comment by timoteialbu** [comment]
Any update on this feature? This is the main reason I've stuck with v4.7 till now.
